### PR TITLE
fix: make Hamburg a separate state

### PR DESCRIPTION
**Pull request by LEQADA**
Hamburg is now a separate state and not a part of Bavaria :slightly_smiling_face:

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Comment by LEQADA** [comment]
@MaledongGit thank you for review. Should we merge it now?

**Review by aymen94 — APPROVED** [review]

**Review by Aissaoui-Ahmed — APPROVED** [review]

**Review by lojzatran — APPROVED** [review]

File: locale/en/get-involved/node-meetups.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ REQUIREMENTS
 - Organizer name - Valentin Huber
 - Organizer contact info - [Email](mailto:valentin.huber@msg.group)
 
-##### Hamburg
+#### Hamburg
```
Comment (heshamMassoud) [inline locale/en/get-involved/node-meetups.md]:
```suggestion
#### Hamburg 🍔
```

Comment (lojzatran) [reply locale/en/get-involved/node-meetups.md]: Why do you think so?

Comment (heshamMassoud) [reply locale/en/get-involved/node-meetups.md]: I think why not? is the question..

Comment (lojzatran) [reply locale/en/get-involved/node-meetups.md]: Hm I've never thought that way...

**Review by heshamMassoud — CHANGES_REQUESTED** [review]
👍

**Review by heshamMassoud — APPROVED** [review]

**Review by JudeNiroshan — APPROVED** [review]
good job @LEQADA !

**Pull request merged**